Refactor examples and modules with run command flag change (#18708)
ballerina-platform/ballerina-lang: stdlib/config-api/src/main/ballerina/src/config/Module.md

> The `ballerina/config` module provides the Config API to read configurations from environment variables, TOML files, and command-line parameters, and build a consolidated set of configurations.

Reviewer: ```suggestion
The `ballerina/config` module provides the Config API to read configurations from environment variables, TOML files, and command-line parameters and build a consolidated set of configurations.
```

The precedence order for configuration lookup is as follows: 

> 1. CLI configurations (used with the --)

Reviewer 2: This should actually be CLI arguments right? CLI configurations sound like it's configs for the CLI

Author: +1 Replaced configurations with arguments.

### Setting configurations

> To explicitly specify a configuration file, the `--b7a.config=<path to configuration file>` property can be used. If 

Reviewer: ```suggestion
To specify a configuration file explicitly, the `--b7a.config=<path to configuration file>` property can be used. If 
```

this property is not set when 

> running a project, Ballerina looks for a `ballerina.conf` file in project root. When running a single file or a `.balx`, it's picked from the same directory as the `.balx` or source. The path to the configuration file can either be an absolute or a relative path. 

Reviewer: ```suggestion
running a project, Ballerina looks for a `ballerina.conf` file in the project root. When running a single file or a `.balx`, it's picked from the same directory in which the `.balx` or source resides. The path to the configuration file can either be an absolute or a relative path. 
```
